tests/locale/keymap_or_locale.pm

```diff
@@ -8,13 +8,13 @@
 # without any warranty.
 
 # Summary: Keyboard layout test in console and display manager after boot
-# Maintainer: Martin Loviska <mloviska@suse.com>
+# Maintainer: Oliver Kurz <okurz@suse.de>
```

Comment:
@mloviska ok with this as discussed lately?

Reply:
yeah, sure